### PR TITLE
Parse contrib aff text if no other tags are specified.

### PR DESCRIPTION
**Pull request by gnott**
To support the Crossref library to extract contributor affiliations when they are as a text string only, this will set the text attribute. It can be used further downstream in the Crossref library. It shouldn't have any impact on other systems.

### CODE REVIEW

File: elifetools/parseJATS.py [hunk 1]
```diff
@@ -1078,7 +1078,7 @@ def format_contributor(contrib_tag, soup, detail="brief", contrib_type=None,
             contrib_affs = {}
             (none_return, aff_detail) = format_aff(aff_tag)
             if len(aff_detail) > 0:
```
Comment (seanwiseman) [inline elifetools/parseJATS.py]: Can be if aff_detail: an empty list evaluates falsely, replicates the comparison being made.

Comment (gnott) [reply elifetools/parseJATS.py]: Here yes I think it can, I am adding a change here too. Some places it looks for a list len it could be related to BeautifulSoup tags, and there I think it can also be changed, where it suggests testing for ``is None``, I think, instead of checking for length. Another day we can do that cleanup too.
